NewDawn/NewDawn/Models/UserModels.swift

```diff
@@ -231,7 +235,10 @@ class UserProfileBuilder{
     
     static func parseProfileInfo(profile_data: [String: Any]) -> NSDictionary {
         let user_data = profile_data["user"] as? NSDictionary
+        let account_data = profile_data["account"] as? NSDictionary
+        let account_id_int = account_data?["id"] as? Int ?? -1
```

Comment:
let's use the id field in user data instead. It should be the ground truth id of unique users.

Reply:
Should I store both account id and user id, since our user action model use account as foreign key?

Reply:
Oh just realize that. let's change user action model to bind to user id instead. Sorry missed that in previous review. Account is a model to keep track of a user's meta info. User is the ground truth representation of real users. We should use User model to link any user actions.

Reply:
OK sure 👌 .